spec/scene-perception.html

```diff
@@ -630,13 +642,13 @@
             The meshing threshold of scene perception.
           </dd>
           <dt>
-            CaptureSize? colorCaptureSize;
+            ImageSize? colorCaptureSize;
```

Comment:
I'm not sure if this name is still suitable here, is it more intuitive to be named as 'colorImageSize'?